compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -131,8 +131,12 @@ object DottyBackendInterface {
 
       def isStaticConstructor(using Context): Boolean = (sym.isStaticMember && sym.isClassConstructor) || (sym.name eq nme.STATIC_CONSTRUCTOR)
 
+      /** Fields of static modules will be static at backend */
+      def isStaticModuleField(using Context): Boolean =
+        sym.owner.isStaticModuleClass && sym.isField && !sym.name.is(LazyBitMapName)
```

Comment:
The special handling of lazy bitmap fields should be explained here (maybe with a reference to the other comment in BCodeSkelBuilder)